### PR TITLE
add dark theme to web3 modal

### PR DESCRIPTION
**Pull request by amochuko**
This PR add dark theme to the web3 modal

## Related GitHub Issue

closes https://github.com/nation3/citizen-app/issues/310

## Screenshots (if appropriate):

_Before_ code change:
> 
before

_After_ code change:
> 
Screenshot 2024-08-26 at 09 49 43

## How Has This Change Been Tested?

- [x] All [status checks](https://github.com/nation3/citizen-app/blob/main/.github/workflows/ui_mainnet.yml#L35) pass (build, lint, e2e, test)
- [x] Works on Sepolia preview deployment
- [ ] Works on Mainnet preview deployment

## Are Any Admin Tasks Required?

- [x] No admin tasks

### CODE REVIEW

**Comment by aahna-ashina** [comment]
@amochuko Your branch was out-of-date, so I updated it:  https://github.com/nation3/citizen-app/pull/311/commits/88ef927d6f528c21bbc8fd7709fab636fba4e512

**Comment by amochuko** [comment]
> @amochuko Your branch was out-of-date, so I updated it: [88ef927](https://github.com/nation3/citizen-app/commit/88ef927d6f528c21bbc8fd7709fab636fba4e512)

I didn't notice it was.
Thanks btw.

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
@amochuko We want the layout in the dark theme to be the same as in the light theme, so please don't add borders to the dark theme.  Only the colors should be different, not the layout.

> 

> 

The [task](https://github.com/nation3/citizen-app/issues/310) is only to add dark theme to the Web3 modal, so your PR includes too many changes.

Please close this PR, and create a new one where you only change these two things:

1. Add dark background color to the Web3 modal.
2. Add light font color to the Web3 modal.

File: ui/components/ErrorCard.tsx [hunk 1]
```diff
@@ -5,7 +5,11 @@ export default function ErrorCard({ error }: any) {
   const { removeError } = useErrorContext()
 
   return (
-    <div className="card shadow-md bg-error text-primary-content">
+    <div
+      className="card shadow-md bg-error text-primary-content
+    dark:border-[1px] dark:border-slate-600 dark:bg-slate-800
+    dark:text-slate-300"
+    >
```
[the remark this answers is not in the document]
Comment (amochuko) [reply ui/components/ErrorCard.tsx:12]: Alright. I'll go make a fix.

**Pull request closed without merging**